tsconfig.json

```diff
@@ -1,15 +1,17 @@
 {
   "extends": "./node_modules/gts/tsconfig-google.json",
   "compilerOptions": {
-    "allowJs": true,
-    "declaration": false, // TODO: Remove when all source are migrated to TS
-    "outDir": "build"
+    "resolveJsonModule": true,
+    "outDir": "build",
+    "lib": [
+      "es2016",
+      "dom"
```

Comment:
Is "dom" needed? I thought that was merely for browser support.

Reply:
Yep. The thing is, we support browser now (kind of). We check for `typeof window !== "undefined"` in the generated client (to enable gRPC-fallback if we are in a browser), and to make it compile, we need `dom`.